apps/bar/bar-api/perftest.yaml

```diff
@@ -6,10 +6,10 @@ spec:
   releaseName: bar-api
   values:
     java:
-      replicas: 2
+      replicas: 1
       autoscaling:
         enabled: true
-        maxReplicas: 2
```

Comment:
why maxReplicas and replicas changed to 1 any reason?

Reply:
This was advised by Dave Jones, he is aware that this means replicas won't be able to scale up or down and he says bar will be decommissioned soon.